frontend/src/metabase/visualizations/echarts/cartesian/option/axis.ts

```diff
@@ -165,7 +154,11 @@ export const buildDimensionAxis = (
         color: getColor("border"),
       },
     },
-  };
+    minInterval:
+      timeSeriesInterval != null
```

Comment:
This is asking ECharts to show ticks with granularity no smaller than the unit size. However, it does not always respect it.